scripts/startTriggers.sh

```diff
@@ -15,28 +15,28 @@ cleanup() {
 trap "cleanup" INT QUIT TERM
 
 if [ $# -lt 2 ]; then
-    echo "${0} SANDBOX_HOST SANDBOX_PORT [DAR_FILE [PARTYID]]"
+    echo "${0} SANDBOX_HOST SANDBOX_PORT [DAR_FILE [USERID]]"
     exit 1
 fi
 
 SANDBOX_HOST="${1}"
 SANDBOX_PORT="${2}"
 DAR_FILE="${3}"
-PARTYID="${4:-BankA}"
+USERID="${4:-banka}"
```

Comment:
Is it `banka` and not `bankA` or doesn't matter?

Reply:
users are always lowercase, so it has to be `banka`